frontend/src/views/Administration.vue

```diff
@@ -164,18 +315,44 @@ export default {
     projectDetails () {
       return getProjectDetails(this.project)
     },
+    memberItems () {
```

Comment:
on the `Members.vue` we named a similar function **user**List.
So the idea was:
- memberList: contains users and service accounts
- userList: only users
- serviceAccountList: only service accounts

Reply:
renamed to `userList`
✔️